implementations/rust/sugar-claim-envelope/src/lib.rs

```
@@ -36,9 +36,8 @@ use serde::{Deserialize, Serialize};
 use serde_json::Value as JsonValue;
 use sugar_canonicalizer::{blake3_512_of, encode_jcs, json_to_value, Value};
 use sugar_proof_envelope::{
-    context_manager_semantics_v1_to_json, ed25519_pubkey_string, ed25519_sign_string,
-    import_signature_v2_to_json, AuthorityMementoRef, ContextManagerSemanticsV1,
-    ContractMementoRef, Ed25519Seed, ImportSignatureV2,
+    ed25519_pubkey_string, ed25519_sign_string, AuthorityMementoRef, ContractMementoRef,
```

Clippy (1.96.0)

[warning] function `encode20` is never used

(warning)